lib/puppetserver/ca/action/generate.rb

```diff
@@ -232,7 +235,7 @@ def generate_certs(certnames, alt_names, settings, digest)
               @logger.inform "Certificate for #{certname} was autosigned."
               true
```

Comment:
Can we add something in here (a warning maybe?) that when the cert is autosigned, the TTL will be ignored?